raster/src/main/scala/geotrellis/raster/rasterize/Rasterizer.scala

```diff
@@ -206,15 +206,14 @@ object Rasterizer {
     * first point to the last point.
     */
   def foreachCellByLineString(line: Line, re: RasterExtent)(f: (Int, Int) => Unit) {
-    val cells = (for(coord <- line.jtsGeom.getCoordinates()) yield {
-      (re.mapXToGrid(coord.x), re.mapYToGrid(coord.y))
-    }).toList
-
-    for(i <- 1 until cells.size) {
-      foreachCellInGridLine(cells(i - 1)._1,
-                            cells(i - 1)._2,
-                            cells(i)._1,
-                            cells(i)._2, line, re, i != cells.size - 1)(f)
+    val coords = line.jtsGeom.getCoordinates()
+    var i = 1; while (i < coords.size) {
```

Comment:
Curious, for my own understanding, why you didn't use `cfor` like other parts of GT?

Reply:
No particular reason, I hope that it isn't too offensive.

Reply:
Nope! Just wondering. No opinion either way, although I do appreciate consistency.